test/cbmc/proofs/Sntp_ReceiveTimeResponse/Makefile

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = Sntp_ReceiveTimeResponse_harness
+
+# Please see test/cbmc/stubs/core_sntp_stubs.c for
+# more information on MAX_NETWORK_RECV_TRIES.
+MAX_NETWORK_RECV_TRIES=5
+
+# Bound on the timeout in Sntp_ReceiveTimeResponse. This timeout is bounded because
+# memory saftey can be proven in a only a single iteration.
+# Each iteration will try to receive a single packet in its entirey. With a time
+# out of 1 we can get coverage of the entire function. Another iteration will
+# performed unnecessarily duplicating of the proof.
+SNTP_RECEIVE_TIMEOUT=1
+
+# Maximum number of sntp time servers
+MAX_NO_OF_SERVERS=5
+
+# Maximum number of attempts in outer loop of Sntp_ReceiveTimeResponse needed to receive a response from server.
+MAX_ITERATIONS_RECEIVE_RESPONSE=2
```

Comment:
Shouldn't this value be set to `1` and be used for the number of loops unwinds for `Sntp_ReceiveTimeResponse` on line 34?